marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/impl/jobs.sjs

```diff
@@ -51,15 +51,20 @@ class Jobs {
 
   getJobDocWithId(jobId) {
     let jobUri = "/jobs/" + jobId + ".json";
-    if (cts.contains(cts.doc(jobUri), cts.jsonPropertyValueQuery("jobId",jobId))) {
-      return this.getJobDocWithUri(jobUri);
-    }
+    return hubutils.queryLatest(function() {
```

Comment:
can you please return fn.head of the query response ? Also in  getJobDocWithUri(jobUri) method